### DISCUSSION TITLE
Welcome to Shuffle-Cybersecurity-Engineer Discussions!

### DISCUSSION THREAD

**Discussion by ashwinexe** [GitHub-Octernships/Shuffle-Cybersecurity-Engineer#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rajeshkanugu** [comment]
may I know what is the difference between ashwinexe-patch-1 and assignment-1 branch

**Reply by ashwinexe** [reply]
It's a redundant branch and has been removed.

**Comment by geekymehta** [comment]
After merging my first pull request from the "sumit-mehta" branch to the main branch, I faced a workflow run failure. The error message stated ".github/workflows/auto-close-pr.yaml: No jobs were run," and this issue persisted with subsequent pull requests merges.

To address this, I followed a suggestion from this discussion to delete the "auto-close-pr.yaml" file in the ".github/workflows" directory. Since then, I haven't seen any workflow run failure messages in GitHub Actions. However, I'm unsure how to confirm whether the workflow runs were successful.

Seeking guidance on this matter.

**Comment by rajeshkanugu** [comment]
Hello ashwin I have completed the assignment and created a pull request and added you as the reviewer is it the correct way to submit or do you want me to make any changes

**Reply by ashwinexe** [reply]
You can merge your PR so the team can start reviewing it :)

**Comment by rajeshkanugu** [comment]
Does every one receive a gmail whether they are selected or not ?

**Reply by ashwinexe** [reply]
Yes.

**Reply by geekymehta** [reply]
when can we expect an email?

**Comment by rajeshkanugu** [comment]
are the results declared ?

**Reply by rajeshkanugu** [reply]
following up on last week's request, are the results declared yet ?

**Reply by ashwinexe** [reply]
Hey, the team is working hard on reviews, we got some amazing submissions. We will create a discussion announcement and also send out emails once Octern(s) are selected.✨
